packages/form-js-viewer/src/render/components/FormField.js

```diff
@@ -43,9 +43,9 @@ export default function FormField(props) {
     throw new Error(`cannot render field <${field.type}>`);
   }
 
-  const value = get(data, _path);
+  const value = get(data, field._path);
```

Comment:
Why is this change necessary?

Reply:
I wanted to homogenize the access, it was weird to me that we did object de-structuring on this one. But I forgot to delete the above.